setup.py

```diff
@@ -38,16 +51,26 @@
 
 sources = ['source/' + x for x in sources]
 
-sources+=['pyqstem/qstem_interface.pyx','pyqstem/QSTEM.cpp']
-
-setup(name='qstem',
-        packages = find_packages(),
-        ext_modules=cythonize(Extension('pyqstem.qstem_interface',
-                    sources=sources,
-                    library_dirs=library_dirs,
-                    libraries=libraries,
-                    include_dirs=include_dirs,
-                    extra_compile_args=['-std=c++11','-D MS_WIN64'],
-                    language='c++')),
+sources += ['pyqstem/qstem_interface.pyx','pyqstem/QSTEM.cpp']
+
+setup(name='pyqstem',
+      packages = find_packages(),
+      version = '1.0',
+      description=description,
+      long_description=long_description,
+      maintainer="Jacob Madsen",
+      maintainer_email="jacob.jma@gmail.com",
+      url="https://github.com/jacobjma/PyQSTEM",
+      license='GPLv3',
+      data_files = [("", ["License.txt"])],
+      platforms = ["Windows", "Linux", "Mac OS-X", "Unix"],
+      install_requires=['matplotlib!=3.0.0', 'ase', 'Cython', 'scikit-image', 'pillow'],
+      ext_modules=cythonize(Extension('pyqstem.qstem_interface',
+                                      sources=sources,
+                                      library_dirs=library_dirs,
+                                      libraries=libraries,
+                                      include_dirs=include_dirs,
+                                      extra_compile_args=['-std=c++11','-D MS_WIN64'],
```

Comment:
Is this -D MS_WIN64 going to do anything silly if it runs on linux/mac?

Reply:
Good catch!  I don't know if it does any harm, but it is certainly wrong.  Also, ``-std=c++11`` cause a warning with the Windows compilers, and does not seem necessary there.